observability/micrometer/src/main/java/org/wildfly/extension/micrometer/MicrometerSubsystemRegistrar.java

```diff
@@ -70,19 +83,21 @@ class MicrometerSubsystemRegistrar implements SubsystemResourceDefinitionRegistr
             "io.micrometer"
     };
 
+    @Deprecated
     public static final SimpleAttributeDefinition ENDPOINT = SimpleAttributeDefinitionBuilder
             .create(MicrometerConfigurationConstants.ENDPOINT, ModelType.STRING)
-            .setAttributeGroup(MicrometerConfigurationConstants.OTLP_REGISTRY)
             .setRequired(false)
+            .addFlag(AttributeAccess.Flag.ALIAS)
             .setAllowExpression(true)
             .setRestartAllServices()
             .build();
 
+    @Deprecated
     public static final SimpleAttributeDefinition STEP = SimpleAttributeDefinitionBuilder
```

Comment:
Given the field is deprecated and there is a replacement, should this add `.setDeprecated(ModelVersion.create(2, 0, 0))`?